Mathlib/Data/Fintype/BigOperators.lean

```diff
@@ -142,46 +141,56 @@ theorem Fintype.card_piFinset [DecidableEq α] [Fintype α] {δ : α → Type _}
 
 @[simp]
 theorem Fintype.card_pi {β : α → Type _} [DecidableEq α] [Fintype α] [f : ∀ a, Fintype (β a)] :
-    Fintype.card (∀ a, β a) = ∏ a, Fintype.card (β a) :=
-  Fintype.card_piFinset _
+    Fintype.card (∀ a, β a) = ∏ a, Fintype.card (β a) := by
+  -- Porting note: term mode proof `Fintype.card_piFinset _` no longer works
+  unfold Fintype.card
+  erw [Fintype.card_piFinset]
+  apply Finset.prod_congr rfl
+  simp [Fintype.card]
```

Comment:
This is because the mathlib3 statement is about `Finset.card` not `Fintype.card`